source/api_documentation/transactions_api/_affiliate_optional_param_table.rst

```diff
@@ -0,0 +1,41 @@
+Additional Feature Parameters
+-----------------------------
+
+..  list-table::
+  :widths: 30 8 40
+  :header-rows: 1
+  :class: parameters
+
+  * - Field
+    - Name in Reports
+    - Description
+
+  * - bundled_destination_ids
+    - Bundled Destination Ids
+    - 
+
+  * - bundled_destination_ids_from_network
+    - Bundled Destination Ids From Network
+    - 
+
+  * - call_outcome_outcome_name_2
+    - Outcome Name
+    - 
```

Comment:
Let's add the outcome name description here [whatever the network side said probably]